corgi/collectors/brew.py

```diff
@@ -697,14 +722,10 @@ def get_component_data(self, build_id: int) -> dict:
             component = self.get_container_build_data(build_id, build)
         elif build_type == self.RPM_BUILD_TYPE:
             component = self.get_rpm_build_data(build_id)
-        elif build_type == self.MAVEN_BUILD_TYPE:
-            component = self.get_maven_build_data(build, build_type_info)
         elif build_type == self.MODULE_BUILD_TYPE:
             component = self.get_module_build_data(build)
-        elif build_type == self.WIN_BUILD_TYPE:
-            component = {}
         else:
-            component = {}
+            return {}
```

Comment:
Can we do ```raise BrewBuildTypeNotSupported()``` in this case?

Reply:
Technically we should never even get to this else branch but then linting complaints that `component` may be undefined. I guess I can change it to raise an exception, to make it more explicit what is happening.